src/presets/netlify/preset.ts

```diff
@@ -22,6 +22,9 @@ const netlify = defineNitroPreset(
       dir: "{{ rootDir }}/.netlify/functions-internal",
       publicDir: "{{ rootDir }}/dist/{{ baseURL }}",
     },
+    prerender: {
+      autoSubfolderIndex: false,
+    },
```

Comment:
We should set this in all three presets: `netlify` (this one), `netlifyEdge`, and `netlifyStatic`. 🙏🏼

Reply:
Added